hack/testing/check-curator.sh

```diff
@@ -57,36 +73,36 @@ twomonthsago=`date -u +"$tf" --date="$(date +%Y-%m-1) -2 months -1 day"`
 # project-prod-4weeksago project-today
 # .operations-2monthsago .operations-today
 
-add_message_to_index() {
-    # index is $1
-    # message is $2
-    message=${2:-"curatortest message"}
-    ca=/etc/fluent/keys/ca
-    cert=/etc/fluent/keys/cert
-    key=/etc/fluent/keys/key
-    url="https://logging-es${ops}:9200/$1/curatortest/"
-    oc exec $fpod -- curl -s --cacert $ca --cert $cert --key $key -XPOST "$url" -d '{
-    "message" : "'"$message"'"
-}'
-}
+TEST_DIVIDER="------------------------------------------"
+
 
-set -- project-dev "$today" project-dev "$yesterday" project-qe "$today" project-qe "$lastweek" project-prod "$today" project-prod "$fourweeksago" .operations "$today" .operations "$twomonthsago" default-index "$today" default-index "$thirtydaysago"
-while [ -n "$1" ] ; do
-    proj="$1" ; shift
-    add_message_to_index "${proj}.$1" "$proj $1 message"
-    shift
-done
+basictest() {
+    ops="$1"
+    set -- project-dev "$today" project-dev "$yesterday" project-qe "$today" project-qe "$lastweek" project-prod "$today" project-prod "$fourweeksago" .operations "$today" .operations "$twomonthsago" default-index "$today" default-index "$thirtydaysago"
+    while [ -n "$1" ] ; do
+        proj="$1" ; shift
+        add_message_to_index "${proj}.$1" "$proj $1 message" $ops
+        shift
+    done
 
-# get current curator pod
-curpod=`oc get pods | awk -v "pat=^logging-curator${ops}-.* Running" '/-deploy/ {next}; /-build/ {next}; $0 ~ pat {print $1}'`
-# show current indices
-echo current indices are:
-oc exec $curpod -- curator --host logging-es${ops} --use_ssl --certificate /etc/curator/keys/ca \
-   --client-cert /etc/curator/keys/cert --client-key /etc/curator/keys/key --loglevel ERROR \
-   show indices --all-indices
-# add the curator config yaml settings file
-curtest=`mktemp --suffix=.yaml`
-cat > $curtest <<EOF
+    # get current curator pod
+    curpod=`oc get pods -l component=curator${ops} | awk '$1 ~ curator && $3 == "Running" {print $1}'`
```

Comment:
Do you really want to only use running ones, and if so should you have a check below to see if this comes back with anything?